File: Codeforces/750A New Year and Hurry.py
# https://codeforces.com/problemset/problem/750/A

n,k = map(int,input().split())
v = 240-k
s=0
c=0
for i in range(n):
    s+=(i+1)*5
    if v>=s:
        c+=1
    else:
        break
print(c)

File: Codeforces/750A_New_Year_and_Hurry.cpp
//  https://codeforces.com/contest/750/problem/A
#include<bits/stdc++.h>
int n,k,ans;
int main(){
	scanf("%d%d",&n,&k);
	k=240-k;
	for(int i=1;i*5<=k;++i){
		++ans;
		k-=i*5;
	}
	printf("%d\n",std::min(n,ans));
	return 0;
}
